src/Adapters/AdapterLoader.js

```diff
@@ -23,6 +23,9 @@ export function loadAdapter(adapter, defaultAdapter, options) {
     if (adapter.default) {
       adapter = adapter.default;
     }
+	if(options.adapter) {
```

Comment:
yeah - its a pain to test locally (windows) - so for tiny changes its easiest to just PR them and let travis let me know if its ok.
just trying a variation, and if its ok I'll add a test.

atm you cant configure a push adapter via string, which is a pain.